hydrus/client/networking/ClientLocalServerResources.py

```diff
@@ -702,7 +702,7 @@ def ParseHashes( request: HydrusServerRequest.HydrusRequest ):
         
         if len( hash_ids_to_hashes ) > 0:
             
-            hashes.extend( hash_ids_to_hashes[ hash_id ] )
+            hashes=[hash_ids_to_hashes[ hash_id ]]
```

Comment:
I think it would be better to do
```suggestion
            hashes.append(hash_ids_to_hashes[ hash_id ])
```
to be inline with what's done above for the single hash case.